src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -99,10 +99,14 @@
     <Compile Include="System\Data\ProviderBase\DbConnectionFactory.cs" />
     <Compile Include="System\Data\ProviderBase\DbConnectionInternal.cs" />
     <Compile Include="System\Data\ProviderBase\DbConnectionPool.cs" />
-    <Compile Include="System\Data\ProviderBase\DbConnectionPoolGroupProviderInfo.cs" />
+    <Compile Include="$(CommonPath)\System\Data\ProviderBase\DbConnectionPoolGroupProviderInfo.cs">
```

Comment:
What is your thinking about pulling these files out of SqlClient into shared? My general preference would be for this PR to clearly not modify SqlClient and ODBC in any way. Such refactoring can always possibly follow up later. I know that I may have actually done this myself 😄 thoughts?

Reply:
OK

Reply:
Don't change the SqlClient.csproj
I had taken a pass at consolidating code for SqlCLient and other data providers in `CommonPath`. However if there is something in SqlClient src folder specifically, then that means that I either didn't get to consolidating the file, or it has a lot of SqlClient specific logic and there was no point  consolidating